src/cache/sqlite.ts

```diff
@@ -141,8 +141,8 @@ class SQLiteAdapter {
   }
 
   removeById(collection, id) {
-    const query = ['DELETE FROM #{collection} WHERE key = ?', [id]];
-    return this.openTransaction(collection, query, null, true)
+    const query = 'DELETE FROM #{collection} WHERE key = ?';
```

Comment:
I misread it this way last time too, but it is different. The way it used to be, isMulti would be true, even though it's a single query (cause the form of a query is an array). `['DELETE FROM #{collection} WHERE key = ?', [id]]` is what a query looks like, according to the sqlite plugin's execSQL() method API. It also could take multiple queries of this type, in an array.

The way it is with the latest changes in this PR, openTransaction() takes a string (the `'DELETE FROM #{collection} WHERE key = ?'` part) and a string for the id parameter. The line you highlighted puts them together in a query with the above mentioned form (except the id isn't an array).

I saw that the case where a string is passed instead of an array is handled in the plugin, but I will make another commit, changing it so an array is passed, since it's better to just comply with the documented API - the guy might change how he handles these and we'd be forced to make the change too. I'd also like to refactor the openTransaction() method since it's 70 lines long and does a lot, but at this point and without tests, it's too risky and with little benefit.